src/assets/stylesheets/shared/_layout.scss

```diff
@@ -1,371 +0,0 @@
-///////////////////////////
-// Utility styles
-///////////////////////////
-
-@mixin padding-x($count:15) {
-  @for $i from 0 through $count {
-    .padding-#{$i} {
-      padding: #{$i}px !important;
-    }
-    .padding-top-#{$i} {
-      padding-top: #{$i}px !important;
-    }
-    .padding-left-#{$i} {
-      padding-left: #{$i}px !important;
-    }
-    .padding-bottom-#{$i} {
-      padding-bottom: #{$i}px !important;
-    }
-    .padding-right-#{$i} {
-      padding-right: #{$i}px !important;
-    }
-  }
-}
-
-@mixin create-padding-x-standard-offset() {
-
-    .padding-standard-offset {
-      padding: $grid-gutter-width/2 !important;
-    }
-
-    .padding-top-standard-offset {
-      padding-top: $grid-gutter-width/2 !important;
-    }
-    .padding-left-standard-offset {
-      padding-left: $grid-gutter-width/2 !important;
-    }
-    .padding-right-standard-offset {
-      padding-right: $grid-gutter-width/2 !important;
-    }
-    .padding-bottom-standard-offset {
-      padding-bottom: $grid-gutter-width/2 !important;
-    }
-}
```

Comment:
Do we want to remove this?

FYI, I'm using the following Less conversion in the patternfly-ng repo.

.margin-loop (@i) when (@i > 0) {
  .margin-v {
    margin: ~"@{i}px" !important;
  }
  .margin-top-@{i} {
    margin-top: ~"@{i}px" !important;
  }
  .margin-left-@{i} {
    margin-left: ~"@{i}px" !important;
  }
  .margin-bottom-@{i} {
    margin-bottom: ~"@{i}px" !important;
  }
  .margin-right-@{i} {
    margin-right: ~"@{i}px" !important;
  }
  .margin-loop(@i - 1);
}
.margin-loop(15);